src/coreclr/jit/codegencommon.cpp

```diff
@@ -2213,6 +2213,12 @@ void CodeGen::genEmitMachineCode()
 //
 void CodeGen::genEmitUnwindDebugGCandEH()
 {
+
+#if defined(TARGET_WASM)
```

Comment:
Why the ifdef? This method should work; if it doesn't, there should be a finer-grained ifdef somewhere on the part that doesn't work.